src/components/TextStyle/TextStyle.scss

```diff
@@ -4,21 +4,21 @@ $code-padding: 0 spacing(extra-tight);
 $code-font-size: 1.15em;
 
 .variationPositive {
-  color: color('green', 'dark');
+  color: var(--p-success-text, color('green', 'dark'));
 }
 
 .variationNegative {
-  color: color('red', 'dark');
+  color: var(--p-critical-text, color('red', 'dark'));
 }
 
 .variationCode {
   position: relative;
   padding: $code-padding;
   border-radius: border-radius();
-  background-color: color('sky', 'light');
+  background-color: var(--p-surface-background, color('sky', 'light'));
```

Comment:
Would this need to become `--p-surface-background` or `--p-surface-foreground` depending on where it's used?

Reply:
I think it should be `--p-surface-foreground-subdued`